invenio_records_resources/services/file_metadata_service.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Record Service API."""
+
+from invenio_records_permissions.policies.records import RecordPermissionPolicy
+
+from ..resource_unit import IdentifiedRecord, RecordSearchState
+from .service import Service, ServiceConfig
+
+
+class FileMetadata:
+    """Fake File metadata resource unit."""
```

Comment:
minor: if its fake for now I wouldn't put it in the docstring cuz we are gonna forget to remove it. I'd say a todo or nothing.

Reply:
Going for `TODO`